src/core/expr/fexpr_nth.cc

```diff
@@ -16,7 +16,7 @@ class FExpr_Nth : public FExpr_Func {
     int32_t n_;
 
   public:
-    FExpr_Nth(ptrExpr&& arg, py::oobj n)
+    FExpr_Nth (ptrExpr&& arg, py::oobj n)
```

Comment:
Only curly brackets should be space padded, see the rest of dt code for a reference.